.changeset/sixty-rivers-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-mesh/cache-upstash-redis': patch
+---
+
+New Upstash Redis implementation
```

Comment:
docs? example?

Reply:
Part of this;<br>[https://github.com/graphql-hive/gateway/issues/576#issue-2823286991](https://github.com/graphql-hive/gateway/issues/576#issue-2823286991)

Reply:
great, thanks!